src/main.rs

```diff
@@ -263,6 +266,11 @@ pub enum Message {
     MouseEnter(pane_grid::Pane),
     Paste(Option<segmented_button::Entity>),
     PasteValue(Option<segmented_button::Entity>, String),
+    Password(String),
```

Comment:
It'd be safer if passwords were stored with https://crates.io/crates/secstr

Reply:
Well, benefit of using secret service, is that is is normally unlocked when the user is logged in. Using other alternatives means you have to enter a password to unlock it. And since the use of passwords in the terminal is fairly rare, that would mean that you would have to unlock this almost every time you would use it, which would cancel out any benefits.

Reply:
Deleted my previous comment, since I misunderstood....

But, yes, that sounds like a nice improvement. Will put that on the list.

Reply:
Looking at the code now, that is not actually a password, but the identifier to the password that is retreived and sent to the terminal.